deploy/test/eunomia-rbac/templates/role-operator.yaml

```diff
@@ -0,0 +1,55 @@
+{{- with .Values.eunomia }}
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: eunomia-operator
+  namespace: "{{ .namespace }}"
+rules:
+# leader election   
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - pods
+  verbs:
+  - "*"  
+#Metrics  
+- apiGroups:
+  - ""
+  resources:
+  - services
+  verbs:
+  - "*"
+#Metrics  
+- apiGroups:
+  - "apps"
+  resources:
+  - replicasets
+  - deployments
+  verbs: 
+  - "*"
+# In Kubernetes <1.16, "deployments" also require API group "extensions" (see: https://stackoverflow.com/a/57854939)
+- apiGroups:
+  - "extensions"
+  resources:
+  - deployments
+  verbs:
+  - "*"
+# Metrics  
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - "get"
+  - "create"
+# Metrics  
+- apiGroups:
+  - apps
+  resources:
+  - deployments/finalizers
+  resourceNames:
+  - eunomia-operator
+  verbs:
+  - "update"
+{{- end }}
```

Comment:
How does this file differ from the already existing file `deploy/helm/eunomia-operator/templates/role-operator.yaml`? why can't we reuse that existing one and we have to create a new very similar (or is it identical?) file?

Reply:
I don't think we should have special RBAC that is only deployed when running the tests. We should just you our standard RBAC configs when running the tests.

Reply:
This is the standard test RBAC, but per namespace. Until now it was being deployed each time to the [sole test namespace `test-eunomia-operator`](https://github.com/KohlsTechnology/eunomia/blob/master/scripts/e2e-test.sh#L52) with the use of [`helm template`](https://github.com/KohlsTechnology/eunomia/blob/master/scripts/e2e-test.sh#L50-L52):
- https://github.com/KohlsTechnology/eunomia/blob/master/deploy/helm/eunomia-operator/templates/role-operator.yaml
- https://github.com/KohlsTechnology/eunomia/blob/master/deploy/helm/eunomia-operator/templates/service-account-operator.yaml

We had these in only one test namespace (because all tests were run in only one). Now with each test function being run in a separate test namespace I need the Role, ServiceAccount, and RoleBinding in each namespace for the tests not to raise access errors.

Reply:
@akavel I'll try to reuse the already existing files

Reply:
They are now being reused: 22073da.
Later, I'll squash it and edit this message too.